code/modules/mob/living/carbon/xenomorph/Powers.dm

```diff
@@ -1628,13 +1630,18 @@
 			if("Bloody Larva") // all larva are caste = blood larva
 				if(leader == "") larva_list += xenoinfo
 				larva_count++
+			if("Necromorph")
+				if(leader == "")
```

Comment:
necromorphs should never be able to be made leaders

Reply:
True, it doesn't make sense from a lore perspective...

Reply:
#961

Reply:
check isnt needed